test/test.js

```diff
@@ -427,6 +427,48 @@ describe('Posts', () => {
     })
   })
 
+  it('POST /posts/like/:id | Should like a post by id', (done) => {
+    const user = {
+      username: 'brando',
+      password: 'brando',
+      email: 'brando@brando.com',
+      name: 'Brandon Danis'
+    }
+
+    const user2 = {
+      username: 'brando2',
+      password: 'brando2',
+      email: 'brando2@brando.com',
+      name: 'Brandon Danis'
+    }
+
+    const post = {
+      url: 'myurl.png',
+      description: 'Test Desc',
+      tags: 'Tags'
+    }
+
+    AddUser(user, () => {
+      AddUser(user2, () => {
+        LoginUser(user, (token) => {
+          LoginUser(user2, (token2) => {
+            SubmitPost(post, token, (postInfo) => {
```

Comment:
why is there callback hell?

Reply:
props on adding tests though

Reply:
Callback hell being fixed in separate PR

Reply:
pretty sure you should fix the related callback hell here (for the test in question)

Reply:
It would require major refactoring to remove callback hell

Reply:
I see, ok then

Reply:
Which is what PR #27 is for 😃

Reply:
Yeah it fine since he's fixing it in another PR. Don't start refactoring here